h2/src/main/org/h2/engine/Session.java

```diff
@@ -945,6 +945,17 @@ public void addLock(Table table) {
         locks.add(table);
     }
 
+    /**
+     * Register table as updated within current transaction.
+     *
+     * @param table the table to register
+     */
+    public void registerTable(Table table) {
```

Comment:
can we use something more explicit that "register", that does not tell me much about what this method does. registerTableAsUpdated/registerTableAsLocked ?